terraform/aks/output.tf

```diff
@@ -0,0 +1,3 @@
+output "url" {
+  value = "https://${module.web_application.hostname}/"
```

Comment:
The `url` output may be used here for simplicity (https://github.com/DFE-Digital/terraform-modules/blob/main/aks/application/outputs.tf#L5)

Reply:
The output.tf is created by the new service template. Not sure what to change here?

Reply:
updated